streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -470,6 +470,22 @@ public void shouldResetToDefaultIfRestoreConsumerAutoCommitIsOverridden() {
         assertEquals("false", consumerConfigs.get(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG));
     }
 
+    @Test
+    public void shouldResetToDefaultIfConsumerGroupProtocolIsOverridden() {
+        props.put(StreamsConfig.consumerPrefix(ConsumerConfig.GROUP_PROTOCOL_CONFIG), "classic,consumer");
```

Comment:
nit: "classic,consumer" is not a valid value for the config is it? THat's a bit confusing, maybe I would just use "consumer"

Reply:
Hello @lucasbru,

Thanks for the review 🙇🏼.

In this test case, we aim to ensure that the group protocol is not overridden. The value "classic,consumer" is used here to verify that even if the user provides a valid protocol (valid in a common consumer config, but not in Kafka Streams), it will still be reset to "classic".

If this might cause confusion, I think adding a comment to clarify would be a good idea.
WYDT ?

Reply:
You are saying that "classic,consumer" is a valid value in a common consumer config, but I think it is not. That's why I would not put it here.

Reply:
Hello @lucasbru

Thank you for pointing that out. After double-checking the documentation, I realized that you’re absolutely right—"classic,consumer" is not a valid value in a common consumer config. I appreciate your attention to detail.

I’ll update the test case. Thanks again for your valuable feedback! 🙇🏼